context/defaultClientEntry.js

```diff
@@ -0,0 +1 @@
+module.exports = 'main';
```

Comment:
Why do we need this? Feel like this could use a comment.

Reply:
Well it's shared in 3 places now. Felt like it should be centrally defined.